modules/farm_fd2_school/src/entrypoints/FD1/App.vue

```diff
@@ -167,14 +168,35 @@ export default {
       );
     },
   },
-  methods: {
-    resetForm() {
-      this.date = '2019-06-15';
-      this.crop = null;
-      this.pickedPlant = null;
-      this.quantity = 1;
-      this.unit = null;
-      this.comment = '';
+  watch: {
+    async selectedType() {
```

Comment:
This `watch` can be on the `crop` property that is bound to the "Crop" select.  That way when the select changes, which changes `crop`, this code will run.